gradle-incremental-configuration-cache/src/test/groovy/com/palantir/gradle/configcache/incremental/ValidateConfigurationCacheTaskTest.groovy

```diff
@@ -0,0 +1,97 @@
+/*
+ * (c) Copyright 2025 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.gradle.configcache.incremental
+
+import com.palantir.gradle.plugintesting.ConfigurationCacheSpec
+import org.gradle.testkit.runner.TaskOutcome
+
+class ValidateConfigurationCacheTaskTest extends ConfigurationCacheSpec {
```

Comment:
Can we make sure to call these `IntegrationSpec` rather than `Test`? This is the pattern we normally use for spock based tests across our existing plugins.